tka/aum.go

```diff
@@ -70,6 +71,16 @@ const (
 	//
 	// Only the State optional field may be set.
 	AUMCheckpoint
+	// A AddDenylistNodeKey AUM describes the addition of a node key to
+	// the denylist.
+	//
+	// Only the NodeKey optional field may be set.
+	AUMAddDenylistNodeKey
+	// A RemoveDenylistNodeKey AUM describes the removal of a node key from
+	// the denylist.
+	//
+	// Only the NodeKey optional field may be set.
+	AUMRemoveDenylistNodeKey
```

Comment:
Would it be better to make the deny list append-only? I guess that makes the limit problematic.

Reply:
The larger the state gets, the larger each checkpoint AUM will get, which will not trend in ideal directions. I think in general we should try and keep append-only data out of TKA state, but I can see how a revocation list is kind of hard to clean up.

Reply:
Honestly the more I think this through, the more I'm feeling like a revocation list maybe doesn't really belong in the TKA state; it doesn't fit the data model very well.

If we're OK with retaining the extra complexity for a feature we might not use in future, I guess leaving this with the 64k max is maybe not horrible? Alternatively, rather than allowing selective removal maybe only support add and clear, or even simply "replace"? I'm having a hard time coming up with a reason to remove a single key, whereas if the signing keys are all rotated out, then "clear" seems reasonable enough.